lib/l10n/shipanther_localization.dart

```diff
@@ -174,6 +174,36 @@ class ShipantherLocalizations {
         desc: 'Reset password message',
         locale: locale.toString(),
       );
+  String get profile => Intl.message(
+        'Profile',
+        name: 'profile',
+        desc: 'profile',
+        locale: locale.toString(),
+      );
+  String get save => Intl.message(
+        'Save',
+        name: 'save',
+        desc: 'save',
+        locale: locale.toString(),
+      );
+  String get passwordUpdateError => Intl.message(
+        'Password does not match',
+        name: 'passwordUpdateError',
+        desc: 'Password does not match',
+        locale: locale.toString(),
+      );
+  String get username => Intl.message(
+        'Username: ',
```

Comment:
Why use the `:`. We don't use it anywhere. Text should be consistent across the board